lib/licenses.nix

```
@@ -556,6 +561,11 @@ in mkLicense lset) ({
     fullName = "Imlib2 License";
   };
 
+  info-zip = {
+    spdxId = "Info-ZIP";
+    fullName = "Info-ZIP License";
```

```suggestion
    fullName = "Info-ZIP License";
    url = "http://www.info-zip.org/license.html";
```

Or `https://spdx.org/licenses/Info-ZIP.html`